Source/Widgets/SceneModePicker/SceneModePicker.css

```diff
@@ -34,6 +37,7 @@ span.cesium-sceneModePicker-wrapper {
 
 .cesium-sceneModePicker-wrapper .cesium-sceneModePicker-dropDown-icon {
     -moz-box-sizing: border-box;
+    -webkit-box-sizing: border-box;
```

Comment:
While you are certainly right, until we have a tool that let's you set a minimum version, I think we are better off just using the generated prefixes instead, otherwise it's just going to put them back the next time we run it.